File: tests/machine/x/clj/python_math.clj
(ns main)

(declare r area root sin45 log_e)

(defn -main []
  (def r 3.0) ;; float
  (def area (* Math/PI (Math/pow r 2.0))) ;; float
  (def root (Math/sqrt 49.0)) ;; float
  (def sin45 (Math/sin (/ Math/PI 4.0))) ;; float
  (def log_e (Math/log Math/E)) ;; float
  (println "Circle area with r =" r "=>" area)
  (println "Square root of 49:" root)
  (println "sin(π/4):" sin45)
  (println "log(e):" log_e)
)

(-main)

File: archived/tests/transpiler/x/kt/python_math.kt
fun main() {
    val r: Double = 3.0
    val area: Double = (kotlin.math.PI as Number).toDouble() * (Math.pow(r, 2.0) as Number).toDouble()
    val root: Double = kotlin.math.sqrt(49.0)
    val sin45: Double = kotlin.math.sin((kotlin.math.PI as Number).toDouble() / 4.0)
    val log_e: Double = kotlin.math.ln(kotlin.math.E)
    println(listOf("Circle area with r =", r, "=>", area).joinToString(" "))
    println(listOf("Square root of 49:", root).joinToString(" "))
    println(listOf("sin(π/4):", sin45).joinToString(" "))
    println(listOf("log(e):", log_e).joinToString(" "))
}
